lnwallet/channel.go

```diff
@@ -4178,7 +4171,9 @@ func (lc *LightningChannel) ReceiveHTLC(htlc *lnwire.UpdateAddHTLC) (uint64, err
 // creating the corresponding wire message. In the case the supplied preimage
 // is invalid, an error is returned. Additionally, the value of the settled
 // HTLC is also returned.
-func (lc *LightningChannel) SettleHTLC(preimage [32]byte, htlcIndex uint64) error {
+func (lc *LightningChannel) SettleHTLC(preimage [32]byte,
```

Comment:
The `godoc` comment should be updated to describe the semantics of the additional refs, and also the close key.